packages/rum-core/src/performance-monitoring/breakdown.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2017-present, Elasticsearch BV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+import { getDuration } from '../common/utils'
+import { PAGE_LOAD } from '../common/constants'
+
+/**
+ * Page load transaction breakdown timings
+ *
+ * Interested events from the Navigation timing API
+ */
+const pageLoadBreakdowns = [
+  ['fetchStart', 'requestStart', 'Network'],
```

Comment:
yes, you are right. Network = DNS + TLS + HTTP time - But I was not sure how to name that since we include the HTTP time in TTFB. 

Redirects is separate beast, There are some problems with capturing those metrics 
+ We need an opt in from previous document that redirects to the current one -> Timing Allow Origin Response header must be present. 
+ If there are multiple chains before loading the document, it will be always set to 0. 

`www.a.com` -> `b.a.com` our redirect time would be lost here. I couldnt find any good read on this other than this issue https://github.com/w3c/navigation-timing/issues/73

Reply:
Everything has been moved under Network for now.